generators/app/templates/package-scripts.js

```diff
@@ -1,4 +1,4 @@
-const npsUtils = require('nps-utils')
+const npsUtils = require('nps-utils') // eslint-disable-line
```

Comment:
sorry about my comments😢  I think I'm not in full of concentration here, midnight in Indonesia ✌️ 😄